src/views/design/ScriptDetail/DetailActions.tsx

```diff
@@ -136,34 +145,45 @@ function DetailActions({
                         </>
                     ) : (
                         <>
-                            <Tooltip
-                                title={translator('scripts.detail.main.actions.delete')}
-                                enterDelay={1000}
-                                enterNextDelay={1000}
-                            >
-                                {DeleteButton}
-                            </Tooltip>
-                            <Tooltip
-                                title={translator('scripts.detail.main.actions.report')}
-                                enterDelay={1000}
-                                enterNextDelay={1000}
-                            >
-                                {ReportButton}
-                            </Tooltip>
-                            <Tooltip
-                                title={translator('scripts.detail.main.actions.execute')}
-                                enterDelay={1000}
-                                enterNextDelay={1000}
-                            >
-                                {ExecuteButton}
-                            </Tooltip>
-                            <Tooltip
-                                title={translator('scripts.detail.main.actions.export')}
-                                enterDelay={1000}
-                                enterNextDelay={1000}
-                            >
-                                {ExportButton}
-                            </Tooltip>
+                            {checkAuthority(SECURITY_PRIVILEGES.S_EXECUTION_REQUEST_WRITE, 'PUBLIC')
```

Comment:
Can we already use the security group name  of the script in question instead of the fixed value 'PUBLIC'